crates/mysten-util-mem/src/external_impls.rs

```diff
@@ -13,6 +13,9 @@ malloc_size_of_is_0!(fastcrypto::bls12381::min_sig::BLS12381AggregateSignature);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381PublicKey);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381Signature);
 malloc_size_of_is_0!(fastcrypto::bls12381::min_pk::BLS12381AggregateSignature);
+malloc_size_of_is_0!(fastcrypto::unsecure::signature::UnsecurePublicKey);
```

Comment:
Let's rename this actually to Insecure from Unsecure (irrelevant to that draft PR)